scripts/kconfig/CMakeLists.txt

```diff
@@ -32,7 +32,6 @@ endif()
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake)
 
 find_package(Curses REQUIRED)
-find_package(Intl REQUIRED)
```

Comment:
is this really not required at all or should we disable it only if(MSYS) ?

Reply:
As a general rule, I would prefer to avoid doing things differently across platforms unless there is a good reason to do so.

I looked up what Intl did, and it did not seem important for Zephyr's Kconfig. It also looks like
Zephyr's kconfig builds fine without it.